content/documentation/getting-started.md

```diff
@@ -10,127 +10,61 @@ menu:
     parent: Documentation
 ---
 
-## 1. Download Tegola
+The quickest way to your first vector tile server is via using this
+example [repository](https://github.com/iwpnd/tegola-example-bonn).
+At the end of this exercise you will have an understanding of how
+to setup and configure Tegola. You will also have created your first
+client application utilizing vector tiles.
 
-### Executable
-[Choose the binary](https://github.com/go-spatial/tegola/releases) that matches the operating system Tegola will run on. Find the Tegola file that was downloaded, unzip it, and move it into a fresh directory. Rename this file `tegola`.
+Let's get started!
 
-### Docker Image
-Tegola provides an official Docker release with support for both PostGIS and GeoPackage data providers.  Use `docker pull gospatial/tegola` to get the latest image.
+## Prerequisites
 
-[Check out the docs on Docker Hub](https://hub.docker.com/r/gospatial/tegola/) for details and examples of using the image.
+- [Install Docker](https://docs.docker.com/get-docker/) for your OS.
+- [Install PSQL](https://www.postgresql.org/download/) for your OS.
```

Comment:
We might be able to drop this and just use the postgis container in the docker compose file. This would reduce one potential dependency issue with mismatched versions.